email_reply_parser/__init__.py

```diff
@@ -449,7 +449,9 @@ def remove_signoff(
 
         if len(signoff_matches_start_positions) > 0 and signoff_matches_start_positions[0] < len(body):
             # If a sign-off was found, cut-off email at starting position
-            body = body[:signoff_matches_start_positions[0]]
+            # If there's one than more matches, go for the last one
+            idx = 0 if len(signoff_matches) <= 1 else -1
+            body = body[:signoff_matches_start_positions[idx]]
```

Comment:
I assume there could be a case where there are multiple threads in an email and this may lead us to include the entire thread?

Could we change the regex here at all to handle this better?

Also, I am surprised we are using this implementation and not the [forked library] we use elsewhere. I would say some of the stuff here is out of date, altthough also risky to switch to this on a Friday!!

Reply:
I should have added this as a "comment" rather than "Request changes"

Reply:
>Also, I am surprised we are using this implementation and not the [forked library] we use elsewhere. I would say some of the stuff here is out of date, altthough also risky to switch to this on a Friday!!

Is this the right one? Are we wrong here?

Reply:
This is what we're installing in the requirements:
git+https://github.com/EdgeTier/email-reply-parser.git

I can try and switch to the original library

Reply:
Apologies I misread things here initially, this makes sense

Reply:
Do we want to use the last match? If it is a big thread will this just not work then? Can we just remove whatever the "Thank you" one is and accept the PS as a limitation for now?